packages/lambda-tiler/src/routes/tile.ts

```diff
@@ -55,7 +59,8 @@ export async function wmts(req: LambdaContext): Promise<LambdaHttpResponse> {
     const provider = await Config.Provider.get(providerId);
     if (provider == null) return NotFound;
 
-    const xml = WmtsCapabilities.toXml(host, provider, tileSets, req.apiKey);
+    const apiKey = Router.apiKey(req);
+    const xml = WmtsCapabilities.toXml(host, provider, tileSets, apiKey!);
```

Comment:
I do not like this `!` but the apiKey is checked before this function is called. Im not sure it needs to exist. I might try and refactor this out